### ISSUE TITLE
Trial of the Champion

### ISSUE THREAD

**Issue by Undergarun** [mangosR2/scriptdev2#183]
1. LFG Reward does not works.
2. Intro of the event Not Yet Implemented!
3. Mounts event Not Yet Implemented!
4. Eadric the Pure http://www.wowhead.com/npc=35119 should cast Hammer of Justice http://www.wowhead.com/spell=66863 to any player in order to interact with Hammer of the Righteous http://www.wowhead.com/spell=66867
5. The Faceroller Achievement http://www.wowhead.com/achievement=3803 does not works. Related to issue 4.
6. Argent Confessor Paletress should be spawned as enemy. Appears as friendly for both factions.
7. Argent Confessor Achievement http://www.wowhead.com/achievement=3802 does not work.
8. The Black Knight spawn event not yet implemented. 
9. I've Had Worse achievement http://www.wowhead.com/achievement=3804 does not works.
10. https://github.com/mangosR2/scriptdev2/issues/156

**Comment by Venom** [comment]
for 6:

Normal
UPDATE `creature_template` SET `faction_A`=14 WHERE  `entry`=34928;
UPDATE `creature_template` SET `faction_H`=14 WHERE  `entry`=34928;

For Heroic 
UPDATE `creature_template` SET `faction_A`=14 WHERE  `entry`=35517;
UPDATE `creature_template` SET `faction_H`=14 WHERE  `entry`=35517;

**Comment by Undergarun** [comment]
Thank, you, i'll commit asap.

**Comment by rsa** [comment]
no. palethress spawned true, but must change faction by scenario. script long ime requires rework...
